test/randindex.jl

```diff
@@ -34,4 +34,9 @@ a3 = [3, 3, 3, 2, 2, 2, 1, 1, 1, 1]
 
 @test randindex(a1, a2) == randindex(a2, a1)
 
+@test randindex(ones(Int, 3), ones(Int, 3)) == (1, 1, 0, 1)
+
+a,b = rand(1:5,100_000), rand(1:5,100_000)
```

Comment:
Yes the problem is this test is too big for 32 bit systems. We can just remove a zero from the length of these vectors and it's fine.

```
a,b = rand(1:5,10000), rand(1:5,10000)
```

I'm not sure if it's worth it to add a check in `confusion`.